PinClustering/PinClustering/Model/Managers/NetworkManager.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  NetworkManager.swift
+//  PinClustering
+//
+//  Created by Norbert Agoston on 06/09/16.
+//  Copyright © 2016 Norbert Agoston. All rights reserved.
+//
+
+import UIKit
+
+class NetworkManager: NSObject {
+    
+    static let sharedInstance = NetworkManager()
+    
+    func httpGet(url: NSURL, callback: (NSArray?, String?) -> Void) {
+        let configuration =  NSURLSessionConfiguration.defaultSessionConfiguration()
+        let session = NSURLSession(configuration: configuration)
```

Comment:
i don't see the point in reinitialising the session each time you call this method since you don't change the configuration. You are using the default one. You may have problems by doing this since your session may dealloc and the call will never terminate.